tests/cephadm/test_validate_osd_pool.py

```diff
@@ -0,0 +1,39 @@
+from ceph.ceph_admin.common import fetch_method
```

Comment:
I think we can use `test_cephadm.py` for command execution, if it is cephadm shell command.